net/uradvd/Makefile

```
@@ -25,6 +25,10 @@ define Package/uradvd/description
 	 Advertise an IPv6 prefix/route via SLAAC.
 endef
 
+define Build/Compile
+	$(call Build/Compile/Default,VERSION=r$(subst ~,-,$(PKG_VERSION)))
+endef
```

```suggestion
MAKE_FLAGS += VERSION=r$(subst ~,-,$(PKG_VERSION))
```

try to simplify it with MAKE_FLAGS